core/src/main/java/com/cloud/agent/api/SecurityGroupRulesCmd.java

```diff
@@ -173,7 +173,7 @@ public String getSecIpsString() {
         final StringBuilder sb = new StringBuilder();
         final List<String> ips = getSecIps();
         if (ips == null) {
-            sb.append("0:");
+            sb.append("0;");
```

Comment:
Shouldn't you use here `RULE_COMMAND_SEPARATOR`?

Reply:
Yes, that would avoid future issues in case of an update on the RULE_COMMAND_SEPARATOR. I will update it soon when having the other raised points addressed as well. Thanks for pointing that!